src/lib/ride.ts

```diff
@@ -1,23 +1,33 @@
 import {db, RideModel} from '../database';
+import {Q} from '@nozbe/watermelondb';
+import EventEmitter from 'events';
 
-export function startRide() {
+const ev = new EventEmitter();
```

Comment:
maybe should call `ev` something a bit more specific like `rideEventEmitter`

Reply:
Yip good call.